addons/website_sale/static/src/js/components/media_dialog/website_sale_image_field.js

```diff
@@ -0,0 +1,26 @@
+import { ImageField, imageField } from '@web/views/fields/image/image_field';
+import { useService } from "@web/core/utils/hooks";
+import { registry } from "@web/core/registry";
+
+export class WebsiteSaleImageField extends ImageField {
+
+    setup() {
+        super.setup();
+        this.props.readonly = false;
```

Comment:
I know this code looks like a hack fix, but it's the only way to make a kanbanRecord not readonly, since we can't mount the `image` widget buttons on a readonly field.